internal/renderers/femtovg/build.rs

```diff
@@ -0,0 +1,19 @@
+// Copyright © SixtyFPS GmbH <info@slint-ui.com>
+// SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-commercial
+
+use cfg_aliases::cfg_aliases;
+
+fn main() {
+    // Setup cfg aliases
+    cfg_aliases! {
```

Comment:
Not an objection to the patch, but while i think cfg_alias is great, having a build.rs just for that might have a significant impact on compile time.  I haven't measured it though. So I don't know if it is worth it or not.

Reply:
There are 7 uses of `#[cfg(use_fontconfig)]`. I could replace those with the full expression, although I think that makes the code less readable. I'll post a patch on top and then we can see.